### PR TITLE
Fixed the double or nothing to be more like the real game

### PR DESCRIPTION
**Pull request by rhydiaan**
Made more like real game the it now doubles every time instead of just multiplying then lowered the wager so less likely to get a massive win.

### CODE REVIEW

File: gambling/bot.py [hunk 1]
```diff
@@ -870,47 +871,53 @@ async def NG_start(self, ctx, arg):
...
-      await self.doubler_choice(ctx, current_wager, current_multiplier)
+      await self.double_choice(ctx, current_wager, current_multiplier)
 
-  async def doubler_main(self, ctx, current_wager, current_multiplier):
-    coin_flip = random.randint(1, 2)
-    if coin_flip == 1:
-      await ctx.send("Win!")
-      current_wager = 50 * current_multiplier
+  async def double_main(self, ctx, current_wager, current_multiplier):
+    if random.randint(1, 2) == 1:
```
Comment (Coombszy) [inline gambling/bot.py:894]: I think we shouldn't have actual 50/50 odds. It should be a teeny bit rigged so people net lose money.

Maybe something like 45/55 odds?

**Review by Coombszy — COMMENTED** [review]
See file comment

File: gambling/bot.py (hunk 1 above)
Comment (rhydiaan) [reply gambling/bot.py]: Good idea, sorted

File: gambling/bot.py [hunk 2]
```diff
@@ -33,6 +33,7 @@ def __init__(self, logger: Logger, config, bank, client: discord.Client, coin_ma
     self.cm = coin_manager
     self.sm = scratch_manager
     self.slots = Slot_Machine()
+    self.double_players = []
```
Comment (Coombszy) [inline gambling/bot.py:36]: My one final thought is that we are storing data in this var. It's only in memory and not being persisted to storage. It's not a major issue and could be ignored for now.

**Review by Coombszy — APPROVED** [review]
See one comment, nothing major (A potential future enhancement)

**Pull request merged**